apps/user/views.py

```diff
@@ -18,7 +18,6 @@
 
 class UserViewSet(viewsets.ModelViewSet):
     permission_classes = (AllowAny,)
```

Comment:
dc aici e AllowAny? mai bine faci invers pui in settings default_permision ISAuthnetificatend si unde iti treb allowany schimbi